tests/lib/journalctl.sh

```diff
@@ -14,6 +14,17 @@ start_new_journalctl_log(){
     else
         echo "$cursor" > "$JOURNALCTL_CURSOR_FILE"
     fi
+    echo "New test starts here - $SPREAD_JOB" | systemd-cat
+    test_id="test-${RANDOM}${RANDOM}"
+    echo "$test_id" | systemd-cat
+    for _ in $(seq 20); do
```

Comment:
Why do we need the for loop? IMO having the for loop shows how this _is_ broken because regular code elsewhere won't have a loop to see if things are logged.

Reply:
Agree with @zyga `journald` is supposed to keep the basic consistency, the log will eventually be flushed to proper store and futher logs that come after will be properly ordered. No need to wait for this here.